src/main/java/com/google/sps/data/Login.java

```diff
@@ -0,0 +1,23 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Login {
```

Comment:
Javadoc for public classes are always required. https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required
You can skip Javadoc only for self-explanatory methods.